frontend/src/features/mapSideBar/MapSideBarContainer.tsx

```diff
@@ -19,14 +19,32 @@ import { PropertyDetailsTabView } from './tabs/propertyDetails/PropertyDetailsTa
  * container responsible for logic related to map sidebar display. Synchronizes the state of the parcel detail forms with the corresponding query parameters (push/pull).
  */
 export const MotiInventoryContainer: React.FunctionComponent = () => {
+  const isMounted = useIsMounted();
```

Comment:
Hey @FuriousLlama it seems like your PR may be pulling in unrelated commits? I could be wrong of course but I don't think this commit is part of the add research file logic? Actually didn't Alejandro originally commit this?

Reply:
Thanks for bringing it up. I think is an issue with git not refreshing the diff if a rebase is done